dev/com.ibm.ws.microprofile.config.1.4_fat/fat/src/com/ibm/ws/microprofile/config14/test/LibertySpecificConfigTests.java

```diff
@@ -51,7 +50,7 @@
      * Test covered in io.openliberty.microprofile.config.2.0.internal_fat.
      */
     @ClassRule
-    public static RepeatTests r = RepeatConfigActions.repeat(SERVER_NAME, Version.CONFIG14_EE7, Version.CONFIG14_EE8);
+    public static RepeatTests r = MicroProfileActions.repeat(SERVER_NAME, MicroProfileActions.LATEST, MicroProfileActions.MP33);
```

Comment:
These tests are only supposed to run against 1.x, and I think `LATEST` would make them run against 2.0

Reply:
yep will change

Reply:
done